src/Types/IDEntityChassis.php

```diff
@@ -52,9 +60,13 @@ public static function normalize($value): ?string
      */
     public function isValid(): bool
     {
-        /** @var ChassisCodeValidatorExtension $validator */
-        $validator = static::getContainer()->make(ChassisCodeValidatorExtension::class);
+        if (\is_string($this->value) && $this->value !== '') {
+            /** @var ChassisCodeValidatorExtension $validator */
+            $validator = static::getContainer()->make(ChassisCodeValidatorExtension::class);
+
+            return \is_string($this->value) && $validator->passes('', $this->value);
```

Comment:
Redundant `\is_string($this->value)`. We are check this condition above.

Reply:
Fixed